cogs/sudo.py

```diff
@@ -374,6 +268,62 @@ async def gpuinfo(self, ctx: Context) -> None:
 
         await ctx.send(embed=embed)
 
+    @sudo.command()
+    @check(is_owner)
+    async def cmduser(self, ctx: Context, user: t.Union[Member, User], *, command: str) -> None:
+        """Run a command as another user."""
+        msg = copy.copy(ctx.message)
+        msg.author = user
+        msg.content = ctx.prefix + command
+
+        new_ctx = await self.bot.get_context(msg)
+        await self.bot.invoke(new_ctx)
+
+    @sudo.command()
+    @check(is_owner)
+    async def install(self, ctx: Context, package: str) -> None:
+        "Install a package"
+        async with ctx.typing():
+            os.system(f"pip install {package}")
+        await ctx.message.add_reaction("✅")
+
+        await ctx.send(f"Installed {package}!")
```

Comment:
This is incredibly unsafe and irrelevant, first of all, `pip` isn't even used, we're running from virtual environment, so if anything it should be `pipenv install`, but in this case, it shouldn't even be here as there's no purpose to it, what advantage does this serve? We will have direct access to the server where the bot is running on, we can install these manually, and we don't even need that as we'll be running in docker which will get updated from `Pipfile` directly from github. It doesn't make sense.
```suggestion
```